cli/filplus.go

```diff
@@ -256,7 +256,7 @@ var filplusListAllocationsCmd = &cli.Command{
 		},
 	},
 	Action: func(cctx *cli.Context) error {
-		if cctx.NArg() > 1 {
+		if cctx.NArg() != 1 {
```

Comment:
this one allows zero as well

Reply:
see `if cctx.NArg() == 1 {` block below that has an early return but behaves differently otherwise